### PR TITLE
Adds Nazareth student commitment

### LINKED ISSUE #567: Add Nazareth's student commitment
The link to the student commitment on the website links to `nazareth-programme` repo which is private, in order for it to be visible to applicants I suggest adding it to this repo.
> Action point from the [August biz meeting](https://github.com/foundersandcoders/nazareth-programme/pull/235/files).

### PR DESCRIPTION
**Pull request by shiryz**
Adding the commitment to the master reference after passing this proposal https://github.com/foundersandcoders/nazareth-programme/issues/213
Relates to https://github.com/foundersandcoders/nazareth-programme/issues/228
Fixes #567

### CODE REVIEW

**Comment by rebeccaradding** [comment]
@shiryz do you want me to merge? Don't see any other reviewers.

**Comment by shiryz** [comment]
@rebeccaradding for some reason (probably my bad connection) it did not request a review from the rest. it needs to be approved by at least 2 maintainers.

File: nazareth_student_commitment.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Founders & Coders Nazareth Student Commitment
+
+#### Student Commitment
+
+Founders & Coders is a not-for-profit endeavour whose main purpose is to provide study and training programmes for the general public and to provide workspace for its students and graduates.
```
Comment (jsms90) [inline nazareth_student_commitment.md]: Double "provide"

> Founders & Coders is a not-for-profit endeavour whose main purpose is to provide study and training programmes for the general public, **including a workspace** for its students and graduates.

?

File: nazareth_student_commitment.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Founders & Coders was founded on the principle of reciprocity. This means that we do not charge you tuition fees, but we do expect you to contribute back to the viability and development of the programme, in particular by mentoring future cohorts for at least two weeks of the following course. When you pursue full-time work, we expect you to consider working for one of our employment partners in the UK or Israel (link to list of employers).
+
+#### Code of Conduct
+
+All members of Founders & Coders are required to agree with our [code of conduct](https://github.com/foundersandcoders/master-reference/blob/master/code_of_conduct.md).
```
Comment (jsms90) [inline nazareth_student_commitment.md]: Maybe it's just me, but "required to agree with" sounds strange

"required to abide by"?

File: nazareth_student_commitment.md (hunk 1 above)
Comment (jsms90) [inline nazareth_student_commitment.md]: Double "provide"

> Founders & Coders is a not-for-profit endeavour whose main purpose is to provide study and training programmes for the general public, **including a workspace** for its students and graduates.

?

File: nazareth_student_commitment.md (hunk 2 above)
Comment (jsms90) [inline nazareth_student_commitment.md]: Maybe it's just me, but "required to agree with" sounds strange

"required to abide by"?

File: nazareth_student_commitment.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+## Founders & Coders Nazareth Student Commitment
+
+#### Student Commitment
+
+Founders & Coders is a not-for-profit endeavour whose main purpose is to provide study and training programmes for the general public and to provide workspace for its students and graduates.
+
+Founders & Coders was founded on the principle of reciprocity. This means that we do not charge you tuition fees, but we do expect you to contribute back to the viability and development of the programme, in particular by mentoring future cohorts for at least two weeks of the following course. When you pursue full-time work, we expect you to consider working for one of our employment partners in the UK or Israel (link to list of employers).
```
Comment (claireinez) [inline nazareth_student_commitment.md]: Should "(link to list of employers)" be an actual link?

Comment (rebeccaradding) [inline nazareth_student_commitment.md]: @claireinez yes, good catch. @shiryz we will need to do this before we can merge.

File: nazareth_student_commitment.md (hunk 1 above)
Comment (rebeccaradding) [inline nazareth_student_commitment.md]: @jsms90 this is based on the London version, which I believe uses the same wording, no?

**Comment by des-des** [comment]
@shiryz @rebeccaradding can we make Nazareth programme public?

File: nazareth_student_commitment.md (hunk 3 above)
Comment (shiryz) [inline nazareth_student_commitment.md]: @rebeccaradding where do you suggest to put this list in? also as a `.md` in the `master-reference` or [link to the website](https://foundersandcoders.com/portfolio/)?
Do we have a ready to copy list somewhere :sweat_smile:?

**Comment by shiryz** [comment]
@des-des not sure tbh, I don't think there's anything extra sensitive in there that can't be public, but again wouldn't be so sure.

**Comment by des-des** [comment]
hnm actually london-programme is private

**Comment by iteles** [comment]
@des-des I don't think `london-programme` should be private. The only reason it is is that no one has had the time to go through the closed issues to triple-check... https://github.com/foundersandcoders/london-programme/issues/199#issuecomment-279452715

**Comment by rebeccaradding** [comment]
@iteles @des-des I opened this issue (https://github.com/foundersandcoders/london-programme/issues/199) back in February, and I'm still keen to make everything possible public. I've added this to my list and I should be able to follow through on the action steps within the next month.

**Comment by des-des** [comment]
@rebeccaradding I cant say I want to do the leg work at this time, but since nazareth-programme might be a good place to start as it is prob smaller then other bigger ones we want to make private.

**Comment by iteles** [comment]
Removing my assignment  until this is ready for review.

**Comment by shiryz** [comment]
Closing this pr since `nazareth-programme` is now public.

**Pull request closed without merging**